src/components/connection_handler/test/connection_handler_impl_test.cc

```diff
@@ -463,10 +485,12 @@ TEST_F(ConnectionHandlerTest, GetConnectionType) {
   AddTestSession();
 
   const DeviceHandle handle = 0;
+  std::string test_mac_address;
   std::string test_connection_type;
-  EXPECT_EQ(0,
-            connection_handler_->GetDataOnDeviceID(
-                handle, NULL, NULL, NULL, &test_connection_type));
+  EXPECT_EQ(
+      0,
+      connection_handler_->GetDataOnDeviceID(
+          handle, nullptr, nullptr, &test_mac_address, &test_connection_type));
```

Comment:
@iCollin this is because we have a separate test for each param. This particular UT verifies app_id param. I removed redundant param to avoid confusions.
See 0b92f71